src/assessor_tools.yml

```diff
@@ -520,8 +521,7 @@
         sqlmap/tarball/master"
         assessment_tool_install_dir: /tools/sqlmap
         # This forces the creation of a Python 3 venv
```

Comment:
The logic in [cisagov/ansible-role-assessment-tool](https://github.com/cisagov/ansible-role-assessment-tool) will not create a Python venv with these variables.

```suggestion
```